src/recordpage.cpp

```diff
@@ -88,8 +89,26 @@ RecordType selectedRecordType(const QScopedPointer<Ui::RecordPage> &ui)
     return ui->recordTypeComboBox->currentData().value<RecordType>();
 }
 
+bool isPerfInstalled()
+{
+    QByteArray envPath = qgetenv("PATH");
```

Comment:
Should probably just use http://doc.qt.io/qt-5/qstandardpaths.html#findExecutable

=> `const bool isPerfInstalled = !QStandardPaths::findExecutable("perf").isEmpty()`